ConsoleApp/Program.cs

```diff
@@ -0,0 +1,41 @@
+﻿internal class Program
+{
+    static void WriteArray(int[] arr)
+    {
+        foreach (int element in arr)
+        {
+            Console.Write(element + " ");
+        }
+        Console.WriteLine();
+    }
+    static int[] BubbleSortInt(int[] arr)
+    {
+        int[] copy = new int[arr.Length];
+        Array.Copy(arr, copy, arr.Length);
+        int temp = 0;
+        for (int i = 0; i < copy.Length; i++)
+        {
+            for (int j = 0; j < copy.Length - 1; j++)
+            {
+                if (copy[j] > copy[j + 1])
+                {
+                    temp = copy[j];
+                    copy[j] = copy[j + 1];
+                    copy[j + 1] = temp;
+                }
+            }
+
+        }
+        WriteArray(copy);
```

Comment:
it's not critical, but imagine you're implementing a library method, and give it to another developer, he expects that your method will just sort the array, but it writes it to console as well :)
methods like that should do only what they pretending to do